sqlmesh/core/engine_adapter/base.py

```diff
@@ -2038,6 +2049,12 @@ def _rename_table(
     ) -> None:
         self.execute(exp.rename_table(old_table_name, new_table_name))
 
+    def _ping(self) -> None:
+        try:
+            self._execute(exp.select(exp.Literal.number(1)).sql(dialect=self.dialect))
```

Comment:
```suggestion
            self.execute(exp.select("1"))
```

Reply:
This was intentional. `execute` always makes sure there's a transaction, while starting a transaction might trigger a ping which would cause an infinite loop.

Reply:
Also this will select a string literal, not a number 1.